src/styles/block__card/page.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
This selector should be placed in `normalize` file

Reply:
ready